### PR TITLE
Add bootstrap modal

### PR DESCRIPTION
**Pull request by BNewing**
Added in modals to one of my website pages! Still needs a bit of work to tweak the bootstrap CSS on the modal buttons, but otherwise works and is readable now after a few alterations @camallen

### CODE REVIEW

File: inmysparetime.html [hunk 1]
```diff
@@ -97,6 +123,35 @@ <h3 align='center'> Coding! </h1>
     <a href="https://twitter.com/beverleynewing" target="_blank">
         <img src="https://simplesharebuttons.com/images/somacro/twitter.png" alt="Twitter" />
     </a>
+
+<script>
```
Comment (camallen) [inline inmysparetime.html]: I would extract the script out to it's own file and require it via the `<script src=''>` tag then look at wiring up the events after the DOM has finished loading via the https://developer.mozilla.org/en/docs/Web/Events/DOMContentLoaded event.

**Comment by camallen** [comment]
Looks good -  try running the https://developers.google.com/speed/pagespeed/ insights tool as well.

File: inmysparetime.html (hunk 1 above)
Comment (BNewing) [inline inmysparetime.html]: I'd forgotten about that! I will do that page separately as a revision exercise on the modal stuff here.

**Comment by BNewing** [comment]
I will now merge this into the main branch and work separately on responsiveness as I think that's what the page analytics tool thingy picked up! @camallen

**Comment by camallen** [comment]
sounds like a plan.

**Pull request merged**